modules/ocf_desktop/files/skel/config/hexchat/servlist.conf

```diff
@@ -0,0 +1,569 @@
+v=2.12.4
+
+N=OCF
+E=UTF-8 (Unicode)
+F=23
+D=0
+S=irc.ocf.berkeley.edu/6697
```

Comment:
I think having `J=#rebuild` would be nice here too so they join the `#rebuild` channel by default too.